src/cow-react/api/gnosisProtocol/api.ts

```diff
@@ -366,8 +366,13 @@ async function _handleQuoteResponse<T = any, P extends FeeQuoteParams = FeeQuote
     throw _handleError(error, response, params, 'QUOTE')
   }
 }
-
-function _mapNewToLegacyParams(params: FeeQuoteParams): QuoteQuery {
+// ETH-FLOW orders require different quote params
+// check the isEthFlow flag and set in quote req obj
+const ETH_FLOW_AUX_QUOTE_PARAMS = {
+  signingScheme: 'eip1271',
```

Comment:
I never told you, but you can also set the "verficationGasLimit"
https://github.com/cowprotocol/services/pull/480/files#diff-f0a015b095aab9c268f63ba350ae52af7da7a000c552ebdc732569b05a29f4cfR344

It's the limit on gas that we spend on verifying the "eip1271" orders in the isValidSignature call. You should be able to set this value to 0 - we will subsidize this small cost for the ethflow for the start.

Reply:
@josojo can we of SHOULD we set this parameter to `0`?